sonic_platform_base/module_base.py

```diff
@@ -258,6 +262,31 @@ def get_state_info(self):
         """
         raise NotImplementedError
 
+    def get_bus_info(self, module_name):
+        """
+        Retrieves the bus information for the specified by "module_name" on a SmartSwitch.
+
+        Returns:
+            Returns the PCI bus information in BDF format like "[DDDD:]BB:SS:F"
+        """
+        raise NotImplementedError
+
+    def pci_detach(self, module_name):
+        """
+        Detaches the DPU PCI device specified by "module_name" on a SmartSwitch.
+
+        Returns: True once the PCI is successfully detached.
+        """
+        raise NotImplementedError
+
+    def pci_reattach(self, module_name):
+        """
+        Rescans and reconnects the DPU PCI device specified by "module_name" on a SmartSwitch.
+
+        Returns: True once the PCI is successfully reconnected.
```

Comment:
Can you also call out the negative case? Also, mention how the platform is informed to recover form this

Reply:
Addressed the comment in https://github.com/sonic-net/sonic-platform-common/pull/504